lib/main.coffee

```diff
@@ -5,22 +5,34 @@ module.exports =
       default: ''
       type: 'string'
       description: 'Leave empty to use bundled'
+    lintInlineJavaScript:
+      type: 'boolean'
+      default: false
+      description: 'Lint JavaScript inside `<script>` blocks in HTML or PHP files'
```

Comment:
Would be a good idea if there is no way to update the grammarScopes on the fly. Isn't there any chance to do this?